config/excel.php

```diff
@@ -292,9 +292,26 @@
         |
         | When exporting and importing files, we use a temporary file, before
         | storing reading or downloading. Here you can customize that path.
+        | permissions is an array with the permission flags for the directory (dir)
+        | and the create file (file).
         |
         */
-        'local_path'          => storage_path('framework/cache/laravel-excel'),
+        'local_path'        => storage_path('framework/cache/laravel-excel'),
+
+        /*
+        |--------------------------------------------------------------------------
+        | Local Temporary Path Permissions
+        |--------------------------------------------------------------------------
+        |
+        | Permissions is an array with the permission flags for the directory (dir)
+        | and the create file (file).
+        | If omitted the default permissions of the filesystem will be used.
+        |
+        */
+        'local_permissions' => [
+            // 'dir'  => 0770,
+            // 'file' => 0640,
```

Comment:
`0644`

Reply:
Suggestions can indeed be changed

Reply:
Any chance you have time to pick up the comments or give me access to your branch for changes?

Reply:
Suggested defaults changed in [92651ba](https://github.com/SpartnerNL/Laravel-Excel/pull/3767/commits/92651ba9d41b0a81305ef3afd21bf657e41fb671)